teamhanko/hanko
# feat(hanko-frontend-sdk): introduce separated frontend sdk package

How to test:
1. Checkout the branch and run `npm run build`
2. Publish to your local npm registry
3. Checkout the `chore-hanko-js-switch-to-new-sdk` branch
4. Install the hanko-sdk from your local npm registry
5. `npm run build` hanko-js and try everything out.
6. It should also be possible to import the package in a react app or include it with a script tag as described in the readme file.

Unit tests will follow.
